### PR TITLE
Pay 3638

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by amitchidrewar1301 — DISMISSED** [review]
Changes look good to me

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -14,7 +14,7 @@ def product = "bar"
 def app = "api"
 
 withPipeline(type, product, app) {
-    installCharts()
+    //installCharts()
```
Comment (aravindan89) [inline Jenkinsfile_CNP:17]: @AnilDhawaleGitHub - Please remove this.

File: api/build.gradle [hunk 2]
```diff
@@ -91,6 +89,11 @@ dependencies {
 
     compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.4'
     compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version:'0.0.4'
+    //Added this for spring boot upgrade as swagger was not working with the latest version.
```
Comment (aravindan89) [inline api/build.gradle:92]: @AnilDhawaleGitHub - Is your application not starting up without these change?

**Review by aravindan89 — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 1 above)
Comment (AnilDhawaleGitHub) [reply Jenkinsfile_CNP:17]: Its Removed

File: api/build.gradle (hunk 2 above)
Comment (AnilDhawaleGitHub) [reply api/build.gradle]: yes its failing with building the application only u can refer build no 60 for this pr

**Review by aravindan89 — DISMISSED** [review]

**Review by amitchidrewar1301 — APPROVED** [review]

**Review by aravindan89 — APPROVED** [review]

**Review by amitchidrewar1301 — APPROVED** [review]

**Pull request merged**